_pkgdown.yml

```
@@ -1,3 +1,17 @@
 url: https://gilead-biostats.github.io/gsm.app/
 template:
   bootstrap: 5
+
+navbar:
+  structure:
+    left: [intro, reference, articles, tutorials, news]
```

```suggestion
    left: [intro, reference, articles, news]
```
We don't have any tutorial content (hopefully this is "yet", but let's not put in an empty section in this PR; we'll add it when it's useful). Overall it looks like this is restating the deffault layout, so I'm pretty sure we just just delete this block entirely, actually.